src/averageAges.js

```diff
@@ -20,6 +20,20 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+
+  const manArr = century
+    ? people.filter(
+      person =>
+        (person.sex === 'm') && (Math.ceil(person.died / 100) === century)
+    )
+    : people.filter(
+      person =>
+        person.sex === 'm'
+    );
+
+  const ages = manArr.map(men => men.died - men.born);
+
+  return ages.reduce((sum, age) => sum + age, 0) / ages.length;
```

Comment:
DRY (Don't Repeat Yourself):
You can create a separate function for this logic and reuse it here